### PR TITLE
feat(FEC-13540): allow passing entry id to start playing the playlist from

### PR DESCRIPTION
**Pull request by lianbenjamin**
### Description of the Changes

- **enhancement**

Add the ability to pass an entryId, to playlist API, which the playlist will start playing from.

### CODE REVIEW

File: src/common/playlist/playlist-manager.ts [hunk 1]
```diff
@@ -84,7 +84,18 @@ class PlaylistManager {
           new FakeEvent(PlaylistEventType.PLAYLIST_LOADED, { playlist: this })
         );
```
Comment (SivanA-Kaltura) [inline src/common/playlist/playlist-manager.ts:85]: looks good, just note the lint errors

**Review by SivanA-Kaltura — DISMISSED** [review]

**Comment by SivanA-Kaltura** [comment]
@lianbenjamin the .idea files were added by mistake ?

**Comment by lianbenjamin** [comment]
@SivanA-Kaltura were added when i merged from master into branch.

File: .idea/.gitignore
Comment (semarche-kaltura) [inline .idea/.gitignore:1]: do we need all those .idea files?

File: coverage/Chrome Headless 119.0.6045.159 (Mac OS 10.15.7)/base.css
Comment (semarche-kaltura) [inline coverage/Chrome Headless 119.0.6045.159 (Mac OS 10.15.7)/base.css:1]: I believe coverage folder should be ignored by git

File: demo/index.html
Comment (semarche-kaltura) [inline demo/index.html:1]: revert changes for demo page

File: demo/style.css [hunk 2]
```diff
@@ -7,3 +7,5 @@ body {
   width: 640px;
   height: 360px;
 }
+
+/*xs - 380 x 209*/
```
Comment (semarche-kaltura) [inline demo/style.css:11]: revert

File: samples/playkit-kaltura-cuepoints.js
Comment (semarche-kaltura) [inline samples/playkit-kaltura-cuepoints.js:1]: revert

File: samples/playkit-kaltura-cuepoints.js.map
Comment (semarche-kaltura) [inline samples/playkit-kaltura-cuepoints.js.map:1]: revert

File: samples/playkit-timeline.js
Comment (semarche-kaltura) [inline samples/playkit-timeline.js:1]: revert

**Review by JonathanTGold — CHANGES_REQUESTED** [review]
you push .idea files , if it intentionally please move it to separate PR and explanations 
thanks

File: samples/playkit-timeline.js.LICENSE.txt
Comment (semarche-kaltura) [inline samples/playkit-timeline.js.LICENSE.txt:1]: revert

File: samples/playkit-timeline.js.map
Comment (semarche-kaltura) [inline samples/playkit-timeline.js.map:1]: revert

File: samples/playkit-ui-managers.js
Comment (semarche-kaltura) [inline samples/playkit-ui-managers.js:1]: revert

File: samples/playkit-ui-managers.js.LICENSE.txt
Comment (semarche-kaltura) [inline samples/playkit-ui-managers.js.LICENSE.txt:1]: revert

File: samples/playkit-ui-managers.js.map
Comment (semarche-kaltura) [inline samples/playkit-ui-managers.js.map:1]: revert

**Pull request closed without merging**